.github/workflows/check.yml

```diff
@@ -17,6 +17,24 @@ jobs:
       - run: npx eslint .
         working-directory: react
 
+  lint-python:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```

Comment:
Just use the docker image!

probably need to add the requirements.txt to the image name

Reply:
is that correct?

Reply:
Yeah LGTM!